### PR TITLE
Json deserializer

### PR DESCRIPTION
**Pull request by lgadde**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/query/lookup/dynamic/schema/DynamicLookupProtobufSchemaSerDe.java [hunk 1]
```diff
@@ -1,31 +1,81 @@
...
+
+    public DynamicLookupProtobufSchemaSerDe(JsonNode coreSchema) throws IOException, Descriptors.DescriptorValidationException {
         this.coreSchema = coreSchema;
+        schemaPathAsMap =  new HashMap<String, String>();
+        DescriptorProtos.FileDescriptorSet set = null;
+        String path ="";
+
+        try {
+            path = getSchema();
+            FileInputStream fin = new FileInputStream(path);
```
Comment (patelh) [inline druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/query/lookup/dynamic/schema/DynamicLookupProtobufSchemaSerDe.java:33]: Do you need to close the stream or is it done by another function below?

Comment (lgadde) [reply druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/query/lookup/dynamic/schema/DynamicLookupProtobufSchemaSerDe.java]: fixed

File: druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/query/lookup/dynamic/schema/DynamicLookupProtobufSchemaSerDe.java [hunk 2]
```diff
@@ -1,31 +1,113 @@
...
+        DescriptorProtos.FileDescriptorSet set = null;
+        FileInputStream fin = null;
+        String path ="";
+
+        try {
+            path = getSchema();
+            fin = new FileInputStream(path);
+            set = DescriptorProtos.FileDescriptorSet.parseFrom(fin);
+            protobufMessageDescriptor = Descriptors.FileDescriptor.buildFrom(set.getFile(0), new Descriptors.FileDescriptor[]{}).getMessageTypes().get(0);
+        }catch (IOException ex){
```
Comment (pranavbhole) [inline druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/query/lookup/dynamic/schema/DynamicLookupProtobufSchemaSerDe.java:38]: can you handle the other Exception ?

Comment (lgadde) [reply druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/query/lookup/dynamic/schema/DynamicLookupProtobufSchemaSerDe.java]: fixed.

**Review by pranavbhole — APPROVED** [review]

**Pull request merged**